lib/parsec/fileparse.py

```diff
@@ -302,7 +302,7 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
                 lineno = None
                 if hasattr(exc, 'lineno'):
                     lineno = exc.lineno
-                elif (isinstance(exc, StandardError) or
+                elif (isinstance(exc, Exception) or
```

Comment:
Is this always True?

Reply:
I'm not sure what the `StandardError` logic was meant to achive.

Reply:
We probably don't need the `or ...` bit here?